internal/api/shared/schemas/base_def_query.cue

```diff
@@ -15,9 +15,9 @@ package base
 
 kind: string
 spec: {
-	datasource?: close({
```

Comment:
Just in case, context for future reference: closing the struct here was preventing to add the level/group info on frontend side, that should be ignored by the backend. 
In the long run we may want to decouple frontend & backend definitions. The day this is done, we can put back the `close()` call here.